openmc/settings.py

```diff
@@ -125,6 +126,8 @@ class SettingsFile(object):
         Coordinates of the lower-left point of the UFS mesh
     ufs_upper_right : tuple or list
         Coordinates of the upper-right point of the UFS mesh
+    resonance_scattering : ResonanceScattering or iterable thereof
```

Comment:
I'm not sure about that. Seem simplest just to enforce that it always be an `Iterable`, but then again I don't know the typical use case for this (*i.e.*, if someone wants to use resonance scattering models, are they only interested in using it for a single nuclide or rather many nuclides).